lib/tools/broccoli/builder.ts

```diff
@@ -9,6 +9,7 @@ import * as destCopyLib from "./node-modules-dest-copy";
 let gulp = require("gulp");
 let vinylFilterSince = require("vinyl-filter-since");
```

Comment:
as it is not used anymore, can we remove this require.
Also please remove the `vinyl-filter-since` dependency from `package.json`